CMakeLists.txt

```diff
@@ -21,6 +22,7 @@ endif()
 option(CUDA    "Enable support for CUDA")
 option(UMESIMD "Enable UME::SIMD backend")
 option(VC      "Enable Vc backend")
+option(LIBMVEC "Enable libmvec backend")
```

Comment:
We can just automatically detect the presence of `libmvec` there's no need for the option here.